src/modules/iotjs_module_websocket.c

```diff
@@ -601,7 +601,8 @@ JS_FUNCTION(ws_receive) {
       if (current_buffer + sizeof(uint16_t) > current_buffer_end) {
         break;
       }
-      payload_len = (uint16_t)(current_buffer[0] << 8 | current_buffer[1]);
+      unsigned char* len_buffer = (unsigned char*)current_buffer;
```

Comment:
The commit message ("converting singed char to unsigned short changes the value") does not describe why was this needed. I guess this is needed because shifting signed values would cause incorrect calculations thus these values should be unsinged?

Reply:
Also there is a bit of style problem: before the `*` there should be a single space (please see travis log: https://travis-ci.org/pando-project/iotjs/jobs/545645417#L841 )